src/renderer/components/+extensions/installed-extensions.tsx

```diff
@@ -0,0 +1,169 @@
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import styles from "./installed-extensions.module.css";
+import React, { useMemo } from "react";
+import { ExtensionDiscovery, InstalledExtension } from "../../../extensions/extension-discovery";
+import { Icon } from "../icon";
+import { List } from "../list/list";
+import { MenuActions, MenuItem } from "../menu";
+import { Spinner } from "../spinner";
+import { ExtensionInstallationStateStore } from "./extension-install.store";
+import { cssNames } from "../../utils";
+import { observer } from "mobx-react";
+import type { Column, Row } from "react-table";
+import type { LensExtensionId } from "../../../extensions/lens-extension";
+
+interface Props {
+  extensions: InstalledExtension[];
+  enable: (id: LensExtensionId) => void;
+  disable: (id: LensExtensionId) => void;
+  uninstall: (extension: InstalledExtension) => void;
+}
+
+function getStatus(isEnabled: boolean) {
+  return isEnabled ? "Enabled" : "Disabled";
+}
+
+export const InstalledExtensions = observer(({ extensions, uninstall, enable, disable }: Props) => {
+  if (!ExtensionDiscovery.getInstance().isLoaded) {
+    return <div><Spinner center /></div>;
+  }
+
+  if (extensions.length == 0) {
+    return (
+      <div className="flex column h-full items-center justify-center">
+        <Icon material="extension" className={styles.noItemsIcon}/>
+        <h3 className="font-medium text-3xl mt-5 mb-2">
+          There are no extensions installed.
+        </h3>
+        <p>Please use the form above to install or drag tarbar-file here.</p>
+      </div>
+    );
+  }
+
+  const filters = [
+    (extension: InstalledExtension) => extension.manifest.name,
+    (extension: InstalledExtension) => getStatus(extension.isEnabled),
+    (extension: InstalledExtension) => extension.manifest.version,
+  ];
+
+  const columns: Column<any>[] = useMemo(
+    () => [
+      {
+        Header: "Name",
+        accessor: "extension",
+        width: 200,
+        sortType: (rowA: Row, rowB: Row) => { // Custom sorting for extension name
```

Comment:
Maybe better to use `lodash/orderBy`?

Reply:
No, `lodash/sortBy` will actually sort array, but we need here to provide sorting function instead. The reason for providing it is need to fetch extension names and compare them. Comparator function is using basic algorithm for string sorting https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#description

Reply:
Second argument supports callback: https://lodash.com/docs/4.17.15#orderBy

Reply:
We are providing a callback here for `react-table` internal sorting. I still don't see how lodash can be used here. We're not sorting any values here, we just telling `react-table` how to do it.